src/platform/plugins/shared/data_view_field_editor/public/components/preview/field_list/field_list_item.tsx

```diff
@@ -143,7 +146,16 @@ export const PreviewListItem: React.FC<PreviewListItemProps> = ({
 
   return (
     <>
-      <EuiFlexGroup className={classes} gutterSize="none" data-test-subj="listItem">
+      <EuiFlexGroup
+        className={classes}
+        // highlights the field using token, TODO: migrate whole SCSS file to emotions
+        css={css`
+          background-color: ${isFromScript ? euiTheme.colors.backgroundBasePrimary : undefined};
+          font-weight: ${isFromScript ? 600 : undefined};
+        `}
```

Comment:
```suggestion
        css={isFromScript ? css`
          background-color: ${euiTheme.colors.backgroundBasePrimary};
          font-weight: 600;
        ` : undefined}
```

We could use a token for the font weight too.